waltz-test/src/main/python/waltz_ducktape/tests/validation/smoke_test.py

```diff
@@ -44,6 +45,15 @@ def test_produce_consume_while_killing_a_server_node(self, num_active_partitions
         self.run_produce_consume_validate(lambda: self.simple_validation_func(validation_cmd, timeout),
                                           lambda: self._kill_a_server_node(num_active_partitions))
 
+    @cluster(cluster_spec=MIN_CLUSTER_SPEC)
+    @parametrize(num_active_partitions=1, txn_per_client=100, num_clients=2, interval=600, timeout=360, interruption_length=10, num_interruptions=3)
+    @parametrize(num_active_partitions=4, txn_per_client=100, num_clients=2, interval=1000, timeout=300, interruption_length=20, num_interruptions=1)
+    def test_produce_consume_client_disconnects(self, num_active_partitions, txn_per_client, num_clients, interval, timeout,
+                                                interruption_length, num_interruptions):
+        validation_cmd = self.client_cli.validate_txn_cmd(num_active_partitions, txn_per_client, num_clients, interval)
+        self.run_produce_consume_validate(lambda: self.simple_validation_func(validation_cmd, timeout),
+                                          lambda: self._disconnect_and_reconnect_client(num_active_partitions, interruption_length, num_interruptions))
```

Comment:
The other PR for server-storage network interruption test seems to be creating a new Class "connection_interruption_storage.py" that includes both the test case and the network interruption logic.

In order to be consistent and to make it easier I would recommend having a separate Class "network_interruption_test.py" or "connection_interruption_test.py” that covers all the scenarios/tests corresponding to the network interruption functionality. So, it will contain test cases for "client-server" and "server-storage" connections under one single file.